bigquery_etl/glean_usage/templates/cross_channel_events_unnested.view.sql

```diff
@@ -0,0 +1,16 @@
+-- Generated via ./bqetl glean_usage generate
+CREATE OR REPLACE VIEW
+    `{{ project_id }}.{{ target_view }}`
+AS
+{% for (dataset, channel) in datasets -%}
+{% if not loop.first -%}
+UNION ALL
+{% endif -%}
+{% if app_name == "fenix" -%}
+SELECT e.* EXCEPT (events) REPLACE(mozfun.norm.fenix_app_info("{{ dataset }}", client_info.app_build).channel AS normalized_channel), event.*
+{% else -%}
+SELECT e.* EXCEPT (events) REPLACE("{{ channel }}" AS normalized_channel), event.*
+{% endif -%}
+FROM `{{ project_id }}.{{ dataset }}.events` e
+LEFT JOIN UNNEST(e.events) AS event
```

Comment:
Why is this a LEFT JOIN? I don't think it makes sense to have rows with empty events, so I would expect a CROSS JOIN.

In practice, it probably doesn't matter because I don't think Glean is supposed to send empty event pings.

Reply:
+1. FWIW I've checked across lots of events tables and never seen an empty events array.